temfpy/uncertainty_quantification.py

```diff
@@ -56,18 +56,28 @@ def eoq_model(x, r=0.1):
     and the setup costs of an order `x[2]`. The annual interest rate `r` is treated as an
     additional parameter.
 
+    .. math::
+        y = \sqrt{\frac{24 x_0 x_2}{r x_1}}
+
+    .. figure:: ../../docs/_static/images/fig-eoq-tradeoff.png
+       :align: center
+
+       Holding `x[1]` and `x[2]` constant, an increase in `x[0]` results in a decrease in
+       the setup cost per unit,
+       but an increase in capital cost increases as the stock of inventory increase.
+
     Parameters
     ----------
     x : array_like
```

Comment:
I just noticed when reviewing the other PR's. Are here clear rules when to add a `.` at the end of a description and when not? Now they always added one and we did not in our two functions.

Reply:
It's not said.
I noticed that almost every example in [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html) has a period, but not all of them.

Reply:
Thanks for following up, then let's have a `.` everywhere. @luward @milakis